src/Sylius/Behat/Context/Api/EmailContext.php

```diff
@@ -51,6 +44,14 @@ public function anEmailWithResetTokenShouldBeSentTo(string $recipient, string $l
         );
     }
 
+    /**
+     * @Then no email should be sent
+     */
+    public function noEmailShouldBeSent(): void
+    {
+        Assert::false($this->filesystem->exists($this->emailChecker->getSpoolDirectory()));
+    }
+
```

Comment:
```suggestion
    /**
     * @Then no email should be sent to :recipient
     */
    public function noEmailShouldBeSentToRecipient(string $recipient): void
    {
        Assert::same($this->emailChecker->countMessagesTo($recipient), $count);
    }
```

Thanks to this change, the step becomes more universal, and being able to be used along the step where we send something to one recipient, but not to another one.